lib/src/io.dart

```diff
@@ -30,14 +30,13 @@ String canonicalize(String path) => _couldBeCaseInsensitive
 
 /// Returns `path` with the case updated to match the path's case on disk.
 ///
-/// This only updates `path`'s basename. It always returns `path` as-is on
-/// operating systems other than Windows or Mac OS, since they almost never use
-/// case-insensitive filesystems.
+/// This always returns `path` as-is on operating systems other than Windows or
+/// Mac OS, since they almost never use case-insensitive filesystems.
 String _realCasePath(String path) {
   // TODO(nweiz): Use an SDK function for this when dart-lang/sdk#35370 and/or
   // nodejs/node#24942 are fixed, or at least use FFI functions.
 
-  if (!_couldBeCaseInsensitive) return path;
+  //if (!_couldBeCaseInsensitive) return path;
```

Comment:
Is this supposed to be removed instead of commented out?

Reply:
No, this was supposed to just be a local test! Good catch.